data/maps/MossdeepCity_SpaceCenter_2F/scripts.inc

```diff
@@ -259,8 +259,11 @@ MossdeepCity_SpaceCenter_2F_EventScript_ChoosePartyForMultiBattle::
 MossdeepCity_SpaceCenter_2F_EventScript_DoStevenMultiBattle::
 	special ReducePlayerPartyToSelectedMons
 	frontier_set FRONTIER_DATA_SELECTED_MON_ORDER
-	setvar VAR_0x8004, SPECIAL_BATTLE_STEVEN
+	setvar VAR_0x8004, SPECIAL_BATTLE_MULTI
 	setvar VAR_0x8005, 0
+	trainerbattle_set_trainer_A TRAINER_MAXIE_MOSSDEEP, MossdeepCity_SpaceCenter_2F_Text_JustWantToExpandLand
+	trainerbattle_set_trainer_B TRAINER_TABITHA_MOSSDEEP, MossdeepCity_SpaceCenter_Text_TabithaDefeat
+	setmultibattlepartner PARTNER_STEVEN
```

Comment:
I know this was a conversation in discord but why has to be this convoluted and the macro `multi_2_vs_2` can't be used instead?